app/react/kubernetes/ingresses/CreateIngressView/IngressForm.tsx

```diff
@@ -370,9 +370,11 @@ export function IngressForm({
                       </div>
                     </div>
 
-                    <p className="vertical-center text-muted small col-sm-12 whitespace-nowrap !p-0">
-                      <Icon icon={Info} mode="primary" size="md" />
-                      <span>
+                    <div className="text-muted col-sm-12 !inline-flex gap-1 !p-0 align-top text-xs">
```

Comment:
It could be worth swapping this out with a texttip, and also adding the same change to EE